.drone/drone.yml

```diff
@@ -250,6 +250,10 @@ platform:
   arch: amd64
   os: linux
 steps:
+- commands:
+  - git fetch --tags
+  image: grafana/alloy-build-image:v0.1.18
+  name: Fetch tags
```

Comment:
@kalleep This change came up after I ran `make drone`. It's because `make drone` wasn't ran in an [earlier PR](https://github.com/grafana/alloy/pull/3543). There's a [doc](https://github.com/grafana/alloy/blob/main/.drone/README.md) on how to do this.

I'd also fogotten to run `make drone` on a change I did a few months ago. It's no big deal, but the drone builds might refuse to run automatically on PRs. They might need to be triggered explicitly until `make drone` is ran on the `main` branch.